frontend/src/app/ui/modals/modal/modal.component.ts

```diff
@@ -13,6 +13,9 @@ export class ModalComponent {
   constructor(public modalService: ModalService) {
   }
 
+  /**
+   * closes modal, if not blocked
```

Comment:
Blocking bedeutet, dass man es nicht schließen darf

Close modal if user is allowed to